packages/server/src/api/middlewares/error.handler.ts

```diff
@@ -29,4 +29,8 @@ export default function errorHandler(
 
     // all other errors should be a 500
     res.status(500).json({ error: internalError });
+
+    // let us know about these 500s
+    // TODO: log stack trace
+    console.error(err.message);
```

Comment:
Should we prefix this with something to make it searchable?

Reply:
Yeah that makes sense. I'm going to switch it to a warn and prefix with `Internal Server Errror`